statsmodels/base/model.py

```diff
@@ -510,6 +510,164 @@ def hess(params, *args):
         mlefit.mle_settings = optim_settings
         return mlefit
 
+    def _fit_zeros(self, keep_index=None, start_params=None,
+                   return_auxiliary=False, k_params=None, **fit_kwds):
+        """experimental fit the model subject to zero constraints
+
+        Intended for internal use cases until we know what we need.
+        API will need to change to handle models with two exog.
+        This is not yet supported by all model subclasses.
```

Comment:
As long as the list is relatively short, should the supported subclasses be listed here?

Reply:
This is a moving target, e.g. NegativeBinomial is correctly handled now. I assume GeneralizedPoisson and NegativeBinomial work because they follow the same pattern but there are not unit tests. 
I have no idea about ZeroInflated, nor whether it works with misc models or user defined subclasses of GenericLikelihoodModel. I assume it does with the latter, but haven't checked yet.